react/features/base/react/constants.js

```diff
@@ -5,3 +5,21 @@
  * everything, such as modal-type of components, or dialogs.
  */
 export const OVERLAY_Z_INDEX = 1000;
+
+/**
+ * The types of the buttons.
+ */
+export const BUTTON_TYPES = {
+    PRIMARY: 'primary',
+    SECONDARY: 'secondary',
+    DESTRUCTIVE: 'destructive'
+};
+
+/**
+ * The modes of the buttons.
+ */
+export const BUTTON_MODES = {
```

Comment:
Drop this?

Reply:
I vote to keep this because I might need it.